test/Libraries/RevitNodesTests/Elements/PathOfTravelTests.cs

```diff
@@ -0,0 +1,180 @@
+﻿using Autodesk.DesignScript.Geometry;
+
+using Revit.Elements;
+using NUnit.Framework;
+
+using RevitTestServices;
+
+using RTF.Framework;
+using Revit.Elements.Views;
+
+namespace RevitNodesTests.Elements
+{
+   [TestFixture]
+   public class PathOfTravelTests : RevitNodeTestBase
+   {
+      [Test]
+      [TestModel(@".\Empty.rvt")]
+      public void Create_ValidArgs()
+      {
+         var pathOfTravelOneToOne = PathOfTravel.ByFloorPlanPoints(
+            GetFloorPlan(),
+            new Point[] { Point.ByCoordinates(0, 0, 0) },
+            new Point[] { Point.ByCoordinates(100, 100, 0) },
+            false);
+       
+         Assert.NotNull(pathOfTravelOneToOne);
+         Assert.Equals(pathOfTravelOneToOne.GetLength(0), 1);
```

Comment:
Here Assert.Equals should not be used for Assertions. Should replace it with Assert.AreEqual like 
Assert.AreEqual(1, pathOfTravelOneToOne.GetLength(0));

Reply:
Thanks, fixed in a new commit.